code/obj/mining.dm

```diff
@@ -193,6 +193,8 @@
 		for (var/turf/T in block(origin, locate(origin.x + width - 1, origin.y + height - 1, origin.z)))
 
 			for (var/mob/living/L in T)
+				if(ismobcritter(L) && isdead(L)) // we don't care about dead critters
+					break
```

Comment:
shouldn't this be `continue` (consider case of dead critter and living human on same tile)

Reply:
[aa2f7e8](https://github.com/goonstation/goonstation/pull/12212/commits/aa2f7e83c6f3b860ca5d9d3fd5c0895eac91391e)